Localization/StringsConvertor/input/ar.lproj/app.json

```diff
@@ -376,7 +382,11 @@
                 "video": "مقطع مرئي",
                 "attachment_broken": "هذا ال%s مُعطَّل\nويتعذَّرُ رفعُه إلى ماستودون.",
                 "description_photo": "صِف الصورة للمَكفوفين...",
-                "description_video": "صِف المقطع المرئي للمَكفوفين..."
+                "description_video": "صِف المقطع المرئي للمَكفوفين...",
+                "load_failed": "Load Failed",
```

Comment:
I'm not aware of the context but maybe `Failed to load` would be more appropriate?